src/Simplex/Chat/Store/Messages.hs

```diff
@@ -1803,3 +1806,18 @@ deleteGroupExpiredCIs db User {userId} GroupInfo {groupId} expirationDate create
   DB.execute db "DELETE FROM messages WHERE group_id = ? AND created_at <= ?" (groupId, min expirationDate createdAtCutoff)
   DB.execute db "DELETE FROM chat_item_reactions WHERE group_id = ? AND reaction_ts <= ? AND created_at <= ?" (groupId, expirationDate, createdAtCutoff)
   DB.execute db "DELETE FROM chat_items WHERE user_id = ? AND group_id = ? AND item_ts <= ? AND created_at <= ?" (userId, groupId, expirationDate, createdAtCutoff)
+
+createChatItemModeration :: DB.Connection -> GroupInfo -> GroupMember -> MemberId -> SharedMsgId -> MessageId -> UTCTime -> IO ()
```

Comment:
```suggestion
createModerationEvent :: DB.Connection -> GroupInfo -> GroupMember -> MemberId -> SharedMsgId -> MessageId -> UTCTime -> IO ()
```

Reply:
+ probably NewModerationEvent

Reply:
we don't have it for reactions either, and this is far more ad-hoc